src/sql/connection/connection.py

```diff
@@ -1010,7 +1025,9 @@ def dialect(self):
     def driver(self):
         return self._driver
 
-    def raw_execute(self, query, parameters=None, with_=None):
+    def raw_execute(
+        self, query, parameters=None, with_=None, enable_named_params="false"
```

Comment:
Please removed `enable_named_params` if we don;'t need it

Reply:
[Removed](https://github.com/ploomber/jupysql/pull/980/commits/9b846cca357c300e2393536b718d03986dd84118#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R1029)